src/components/templates/Audience/Audience.scss

```diff
@@ -34,7 +34,6 @@ $spacing: calc(1 * min(var(--seat-size), var(--seat-size-min)));
     pointer-events: none;
 
     .video-container {
-      margin-top: calc(1 * min(var(--seat-size), var(--seat-size-min)));
```

Comment:
What was the point of adding the top margin?
It looks not broken without it(probably one of the reasons the video overflowed the seats?)

Reply:
Off the top of my head, I feel like it was sitting 'too high' without it, but I don't think I could ever figure out why that was the case.

It could be that some changes we made after I originally added it 'fixed the need for it' too. If it looks fine without it, then it's probably fine to remove it.

It always felt kind of hacky to me.. particularly after we switched to using the flexbox to center it (off the top of my head I was originally positioning it by exact column sizes/etc, so maybe this was leftover from that)

Reply:
Off the top of my head, I feel like it was sitting 'too high' without it, but I don't think I could ever figure out why that was the case.

It could be that some changes we made after I originally added it 'fixed the need for it' too. If it looks fine without it, then it's probably fine to remove it.

It always felt kind of hacky to me.. particularly after we switched to using the flexbox to center it (off the top of my head I was originally positioning it by exact column sizes/etc, so maybe this was leftover from that)

Reply:
I locally changed `VIDEO_MIN_WIDTH_IN_SEATS` back to `8` and `--seat-size-min` back to `25px` for these screenshots.

With reactions enabled:
![image](https://user-images.githubusercontent.com/753891/111413404-5048e080-8732-11eb-82e8-1eae75742076.png)

With reactions disabled:
![image](https://user-images.githubusercontent.com/753891/111413511-7ec6bb80-8732-11eb-94f9-91e781c95b24.png)

I even tried changing it back to a vimeo embed like the old venue, but it still seems fine:
![image](https://user-images.githubusercontent.com/753891/111413772-fb599a00-8732-11eb-9242-a83303815cba.png)
![image](https://user-images.githubusercontent.com/753891/111413853-204e0d00-8733-11eb-82fc-a048406a0606.png)

---

So basically.. no clue why it was there originally/etc, but it seems like it's fine to remove it now.